src/crl.c

```diff
@@ -1258,7 +1258,7 @@ static THREAD_RETURN WOLFSSL_THREAD DoMonitor(void* arg)
     }
 
     if (crl->monitors[1].path) {
-        wd = inotify_add_watch(notifyFd, crl->monitors[1].path, IN_CLOSE_WRITE |
+        wd = inotify_add_watch(notifyFd, crl->monitors[1].path, IN_CREATE |
```

Comment:
Won't this trigger the monitor too quickly? Before the file is fully written?

Reply:
The link() is atomic.

Reply:
I'm worried that this may lead to reading before the file is closed. For link its ok but this can lead to errors in a real scenario.

Reply:
That's the problem I was fixing with the link(). The file is never opened.

Previously, copy_file() would open the source and destination files and copy the source 100 bytes at a time. It was giving the server thread enough time to start up and open the file for reading while it was still getting copied.

link() opens the directory and writes the inode of the source file in, which triggers the create event.

Reply:
But real users are not creating links but copying in actual files containing the CRLs. I'm not convinced that this won't break in their runtime.

Reply:
We can deal with that when a customer brings it up as a problem.

This fixes the problem that the testsuite is failing intermittently in our environments.

Reply:
I don't believe that is the correct approach. We don't want to break user environments.